core/lib/lh-error.js

```diff
@@ -23,6 +23,11 @@ const UIStrings = {
    * @example {NO_FCP} errorCode
    * */
   noFcp: 'The page did not paint any content. Please ensure you keep the browser window in the foreground during the load and try again. ({errorCode})',
+  /**
+   * @description Error message explaining that the Largest Contentful Paint metric was not seen during the page load.
+   * @example {NO_LCP} errorCode
+   * */
+  noLcp: 'The page did not paint anything that qualifies as LCP. Ensure the page has a valid LCP element and then try again. ({errorCode})',
```

Comment:
possible tweaks:

```suggestion
  noLcp: 'The page did not display content that qualifies as a Largest Contentful Paint (LCP). Ensure the page has a valid LCP element and then try again. ({errorCode})',
```

Reply:
should we just delete the `errorCode` part now that it's specific?

Reply:
> should we just delete the errorCode part now that it's specific?

This message is also used for `NO_LCP_ALL_FRAMES`.

Reply:
OTOH we have [pretty good SEO on "NO_LCP", apparently](https://www.google.com/search?q=NO_LCP), so we should probably just leave it :)